code/modules/surgery/organs/augments_internal.dm

```diff
@@ -83,7 +86,7 @@
 
 /obj/item/organ/cyberimp/brain/anti_drop/proc/release_items()
 	for(var/obj/item/I in stored_items)
-		I.flags ^= NODROP
```

Comment:
This is an XOR operation, which means it was supposed to be toggling the implant on and off. Someone rewrote the code not knowing what they were doing and broke it.